po/pt.po

```diff
@@ -0,0 +1,361 @@
+msgid "Align table right"
+msgstr "Alinhar tabela à direita"
+
+#: src/Widgets/ToolBars/TableToolBar.vala:84
+msgid "Justify table"
+msgstr "Justificar tabela"
+
+#: src/Widgets/ToolBars/TableToolBar.vala:86
+msgid "Add"
+msgstr "Adicionar"
+
+#: src/Widgets/ToolBars/TableToolBar.vala:90
+msgid "Add a new table"
+msgstr "Adicionar nova tabela"
+
+#: src/Widgets/ToolBars/TableToolBar.vala:92
+msgid "Delete"
+msgstr "Deletar"
+
+#: src/Widgets/ToolBars/TableToolBar.vala:96
+msgid "Delete selected table"
+msgstr "Deletar tabela selecionada"
+
+#: src/Widgets/ToolBars/TextToolBar.vala:43
+msgid "Paragraph"
+msgstr "Parágrafo"
+
+#: src/Widgets/ToolBars/TextToolBar.vala:44
+msgid "Title"
+msgstr "Título"
+
+#: src/Widgets/ToolBars/TextToolBar.vala:45
+msgid "Subtitle"
+msgstr "Subtítulo"
+
+#: src/Widgets/ToolBars/TextToolBar.vala:46
+msgid "Bullet List"
+msgstr "Lista de Pontos"
+
+#: src/Widgets/ToolBars/TextToolBar.vala:47
+msgid "Numbered List"
+msgstr "Lista numerada"
+
+#: src/Widgets/ToolBars/TextToolBar.vala:48
+msgid "Two-Column"
+msgstr "Duas colunas"
+
+#: src/Widgets/ToolBars/TextToolBar.vala:50
+msgid "Set text style"
+msgstr "Selecionar estilo do texto"
+
+#: src/Widgets/ToolBars/TextToolBar.vala:97
+msgid "Align text left"
+msgstr "Alinhar texto à esquerda"
+
+#: src/Widgets/ToolBars/TextToolBar.vala:98
+msgid "Center text"
+msgstr "Centralizar texto"
+
+#: src/Widgets/ToolBars/TextToolBar.vala:99
+msgid "Align text right"
+msgstr "Alinhar texto à direita"
+
+#: src/Widgets/ToolBars/TextToolBar.vala:100
+msgid "Justify text"
+msgstr "Justificar texto"
+
+#: src/Widgets/ToolBars/TextToolBar.vala:103
+msgid "Increase indent"
+msgstr "Aumentar recuo"
+
+#: src/Widgets/ToolBars/TextToolBar.vala:105
+msgid "Decrease indent"
+msgstr "Diminuir recuo"
+
+#: src/Widgets/ToolBars/TextToolBar.vala:115
+msgid "Insert text"
+msgstr "Inserir texto"
+
+#: src/Widgets/ToolBars/TextToolBar.vala:117
+msgid "Insert link"
+msgstr "Inserir link"
+
+#: src/Widgets/ToolBars/TextToolBar.vala:119
+msgid "Insert image"
+msgstr "Inserir imagem"
+
+#: src/Widgets/ToolBars/TextToolBar.vala:121
+msgid "Insert table"
+msgstr "Inserir tabela"
+
+#: src/Widgets/ActionBar.vala:47
+#, c-format
+msgid "%d character with spaces"
+msgid_plural "%d characters with spaces"
+msgstr[0] "%d caractere com espaços"
+msgstr[1] "%d caracteres com espaços"
+
+#: src/Widgets/ActionBar.vala:61
+#, c-format
+msgid "%d character without spaces"
+msgid_plural "%d characters without spaces"
+msgstr[0] "%d caractere sem espaços"
+msgstr[1] "%d caracteres sem espaços"
+
+#: src/Widgets/PreferenceDialog.vala:27 src/Widgets/TitleBar.vala:62
+#: src/Widgets/TitleBar.vala:68
+msgid "Preferences"
+msgstr "Preferências"
+
+#: src/Widgets/PreferenceDialog.vala:33
+msgid "Document folder location:"
+msgstr "Localização da pasta Documentos:"
+
+#: src/Widgets/PreferenceDialog.vala:37
+msgid "Select Document Folder…"
+msgstr "Selecione a pasta do documento..."
```

Comment:
```suggestion
msgstr "Selecione a pasta do documento…"
```

You may want to use ellipses used in the source string instead of three periods.